chandra_aca/cubic_spline.py

```diff
@@ -0,0 +1,434 @@
+"""
+Provides CubicSpline class for Ska2 environment that has an older
+scipy package.  This was added in scipy 0.18 and can be removed
+if/when scipy is upgraded or transition to Ska3 is completed.
+Note that the scipy upgrade wants to bring numpy along for the ride.
+
+The CubicSpline class here is just copied verbatim from the scipy package.
+"""
```

Comment:
Being pedantic, this deserves a copy of the Scipy license.

Reply:
That's not being pedantic, I was remiss in not doing this to begin with.